src/test/java/teammates/logic/core/FeedbackResponseCommentsLogicTest.java

```diff
@@ -346,6 +353,365 @@ public void testGetFeedbackResponseCommentsForQuestionInSection_withSectionName_
+
+        ______TS("success: giver is student; show comment to comment recipient's team; comment is visible"
+                + "to response recipient's team members");
+        student = dataBundle.students.get("student3InCourse1");
+        studentsInTeam = roster.getTeamToMembersTable().getOrDefault(student.getTeam(), Collections.emptyList());
+        studentsEmailInTeam = studentsInTeam.stream()
+                .map(StudentAttributes::getEmail)
+                .collect(Collectors.toSet());
+        assertTrue(frcLogic.isResponseCommentVisibleForUser("student3InCourse1@gmail.tmt", UserRole.STUDENT,
+                student, studentsEmailInTeam, response, relatedQuestion,
+                commentFromS5C1ToR2Q2S1C1VisibleToResponseReceiverTeamMember));
+
+        student = dataBundle.students.get("student4InCourse1");
+        studentsInTeam = roster.getTeamToMembersTable().getOrDefault(student.getTeam(), Collections.emptyList());
+        studentsEmailInTeam = studentsInTeam.stream()
+                .map(StudentAttributes::getEmail)
+                .collect(Collectors.toSet());
+        assertTrue(frcLogic.isResponseCommentVisibleForUser("student4InCourse1@gmail.tmt", UserRole.STUDENT,
+                student, studentsEmailInTeam, response, relatedQuestion,
+                commentFromS5C1ToR2Q2S1C1VisibleToResponseReceiverTeamMember));
+
+        ______TS("failure: giver is student; show comment to comment recipient's team; comment is not visible"
+                + "to students not from response recipient's team");
+        student = dataBundle.students.get("student1InCourse2");
+        studentsInTeam = roster.getTeamToMembersTable().getOrDefault(student.getTeam(), Collections.emptyList());
+        studentsEmailInTeam = studentsInTeam.stream()
+                .map(StudentAttributes::getEmail)
+                .collect(Collectors.toSet());
+        assertFalse(frcLogic.isResponseCommentVisibleForUser("student1InCourse2@gmail.tmt", UserRole.STUDENT,
+                student, studentsEmailInTeam, response, relatedQuestion,
+                commentFromS5C1ToR2Q2S1C1VisibleToResponseReceiverTeamMember));
+
+        student = dataBundle.students.get("student1InUnregisteredCourse");
+        studentsInTeam = roster.getTeamToMembersTable().getOrDefault(student.getTeam(), Collections.emptyList());
+        studentsEmailInTeam = studentsInTeam.stream()
+                .map(StudentAttributes::getEmail)
+                .collect(Collectors.toSet());
+        assertFalse(frcLogic.isResponseCommentVisibleForUser("student1InUnregisteredCourse@gmail.tmt", UserRole.STUDENT,
+                student, studentsEmailInTeam, response, relatedQuestion,
+                commentFromS5C1ToR2Q2S1C1VisibleToResponseReceiverTeamMember));
+
+        // comment visibility follows feedback question's visibility
+        FeedbackResponseCommentAttributes commentFromS4C1ToR2Q2S1C1VisibilityFollowsQuestion =
```

Comment:
Remove "FromXXXXToXXXX" in all the variable names, those are hard to decipher

Reply:
The intention of doing so is to be consitent with the format of typical data bundle. It is now removed.